pricegraph/src/api/price_estimation.rs

```diff
@@ -65,6 +65,10 @@ impl Pricegraph {
             // liquidity from this transitive order.
             current_exchange_rate < flow.exchange_rate.inverse()
         }) {
+            if flow.is_dust_trade() {
+                continue;
+            }
```

Comment:
Seems like this block appears twice... could this be taken care of inside the while condition?

Reply:
Not in this case either - since it would cause us to stop looping which is not what we want.